### PR TITLE
CCF form final polishing

### PR DESCRIPTION
**Pull request by sbinlondon**
Things addressed:

- Spacing between form fields should be less
- Replace climate initiative image
- On OS, don’t let you get to the second screen without agreeing to TOS - force user to agree to both on the first screen and if moving on to manual verification pass thru that the box has been ticked so we don’t ask again
- Don’t display the CollectiveCover if we move from /create to /apply with manual verification
- Collectives without Host redirect to Collective page with banner linking to financial contributions menu
- Github Repo Entry CSS mobile vs desktop
- Remove strict heights from any buttons
- Adding a background image to the 1-step (climate, community) form

Things added that may be removed:

- Add new financial contributions section to the top of the collective page - an interim solution that isn't the banner, though I can remove what I've done and keep the banner (but change the color) or whatever we decide is best to get this feature ready and rolled out for now

Things not addressed:

- new FAQ module and button styles - as discussed on Slack this is something yet to be implemented by @Betree as he is awaiting something from Cuiki
- Redirect URL? (not in scope)
- changing the color of the success banner (will be addressed separately)

### CODE REVIEW

File: components/collective-page/sections/Contribute.js [hunk 1]
```diff
@@ -160,123 +161,168 @@ class SectionContribute extends React.PureComponent {
...
-                    <Flex justifyContent="space-between" alignItems="center" mb={3}>
-                      <H3 fontSize="H5" fontWeight="600" color="black.700">
-                        <FormattedMessage id="CP.Contribute.Financial" defaultMessage="Financial contributions" />
-                      </H3>
-                      <Box m={2} flex="0 0 50px">
-                        <Chevrons />
-                      </Box>
-                    </Flex>
-                  </ContainerSectionContent>
+        {status === 'collectiveCreated' && hasContribute ? (
```
Comment (kewitz) [inline components/collective-page/sections/Contribute.js]: This ternary looks a bit complex, could we decouple the `status` conditional from the `hasContribute`?

File: components/create-collective/NewCreateCollective.js [hunk 2]
```diff
@@ -71,7 +71,7 @@ class NewCreateCollective extends Component {
     } else if (query.category === 'climate') {
       this.setState({ category: 'climate' });
     } else if (!query.category) {
-      this.setState({ category: null });
+      this.setState({ category: null, tos: null });
```
Comment (kewitz) [inline components/create-collective/NewCreateCollective.js]: Can't we just have `tos: null`  as a default value in the state constructor?

**Comment by znarf** [comment]
For images, check https://github.com/opencollective/opencollective-frontend/pull/3616 to understand how I optimized it.

File: components/collective-page/sections/Contribute.js (hunk 1 above)
Comment (znarf) [reply components/collective-page/sections/Contribute.js]: Ideally, `collectiveCreated` should not impact the content of the page, it should just impact the banner. What if the user is coming back to the page?

File: components/create-collective/sections/ConnectGithub.js [hunk 3]
```diff
@@ -359,19 +359,22 @@ class ConnectGithub extends React.Component {
                   <Link
                     route="new-create-collective"
                     params={{
-                      hostCollectiveSlug: 'opensource',
+                      hostCollectiveSlug: 'opensourceorg',
```
Comment (znarf) [inline components/create-collective/sections/ConnectGithub.js]: opensource

File: components/create-collective/sections/CreateCollectiveForm.js [hunk 4]
```diff
@@ -149,7 +164,7 @@ class CreateCollectiveForm extends React.Component {
 
     return (
       <Flex flexDirection="column" m={[3, 0]}>
-        {host && <CreateCollectiveCover host={host} />}
+        {host && query.hostCollectiveSlug !== 'opensource' && <CreateCollectiveCover host={host} />}
```
Comment (znarf) [inline components/create-collective/sections/CreateCollectiveForm.js]: That's not the ideal solution.

File: components/create-collective/NewCreateCollective.js (hunk 2 above)
Comment (sbinlondon) [reply components/create-collective/NewCreateCollective.js]: Yes - updated

File: components/create-collective/sections/CreateCollectiveForm.js (hunk 4 above)
Comment (sbinlondon) [reply components/create-collective/sections/CreateCollectiveForm.js]: We show the Collective navbar on the form for Collectives with a host, unless the host is the Open Source Collective (becaues they are coming from manual verification). So this makes sense to me in a straightforward way. What would you suggest as an alternative?

Comment (sbinlondon) [reply components/create-collective/sections/CreateCollectiveForm.js]: Have discussed via Slack

**Comment by znarf** [comment]
A bit more feedback!

4) If the host is not open to application, it should not be possible to apply (error message instead of the form). Reference to previous behavior:

Capture d’écran 2020-03-05 à 21 59 58

5) Check the alignment in case we have double check boxes:

Capture d’écran 2020-03-05 à 22 03 36

6) On Collective creation, after redirection, not scrolled to page top.

**Comment by sbinlondon** [comment]
4 resolved, 6 resolved, working on 5

File: components/collective-page/sections/Contribute.js [hunk 5]
```diff
@@ -149,134 +150,181 @@ class SectionContribute extends React.PureComponent {
...
+    2b. not admin + Collective active = normal Contribute section ???
+    3. not admin + Collective not active = display nothing ✅
+    */
 
     const createContributionTierRoute = isEvent
       ? `/${collective.parentCollective.slug}/events/${collective.slug}/edit#tiers`
       : `/${collective.slug}/edit/tiers`;
 
-    if (!hasContribute && !hasOtherWaysToContribute) {
+    if (!isAdmin && !isActive) {
```
Comment (Betree) [inline components/collective-page/sections/Contribute.js]: **[:sparkles: **Enhancement**]** One problem about this is that it is going to hide events for archived collectives and collectives not approved yet.

---

<details>
<summary>More details about this feedback</summary>
<br/>

This feedback categorization is inspired by [Netlify's Feedback Ladders system](https://www.netlify.com/blog/2020/03/05/feedback-ladders-how-we-encode-code-reviews-at-netlify). It includes 5 levels of feedback:

:rotating_light: **Critical blocker**
This feedback blocks all related work, and requires immediate action. 

:no_entry_sign: **Blocker**
This feedback should block the work from moving forward or being approved, but it does not necessarily require immediate action from the team.

:hammer_and_wrench: **Important**
This feedback should not block the work from moving forward or being approved, but it does require future action from the team. This generally means the feedback is not required for the MVP or initial draft of this work, but should be addressed at some point and we should ideally create an issue to track it.

:sparkles: **Enhancement**
This feedback should not block the work from moving forward or being approved. The project lead or owner should consider the feedback and implement or open an issue it if they agree with it.

:ice_cream: **Details & personal preferences**
This feedback should not block the work from moving forward or being approved. The project lead or owner should consider the feedback and implement or open an issue it if they deem it useful. This is also the category for everything that relates to personal preferences or code style.

</details>

**Pull request merged**